pkg/api/batch/operation.go

```diff
@@ -32,6 +32,12 @@ type Operation struct {
 	Patch jsonpatch.Patch `json:"patch"`
 	//HashAlgorithmInMultiHashCode
 	HashAlgorithmInMultiHashCode uint `json:"hashAlgorithmInMultiHashCode"`
+	//The logical blockchain time that this operation was anchored on the blockchain
+	TransactionTime uint `json:"transactionTime"`
```

Comment:
If this is the block number then it should be uint64

Reply:
done